docs/ingestion.md

```diff
@@ -4,7 +4,7 @@ There are two ways to ingest data into Parca.
 
 ## Push based
 
-The recommended way to use for generic profiling is Parca Agent. Parca Agent is an always-on sampling profiler that uses [eBPF](https://ebpf.io/) to capture raw profiling data with very low overhead. It observes user-space, and kernel-space stack traces 100 times per second and builds [pprof](https://github.com/google/pprof) formatted profiles from the extracted data. For in-depth detail and explanation refer to the the [Parca Agent Design](/docs/parca-agent-design) documentation.
+The recommended way to use for generic profiling is Parca Agent. Parca Agent is an always-on sampling profiler that uses [eBPF](https://ebpf.io/) to capture raw profiling data with very low overhead. It observes user-space, and kernel-space stack traces 19 times per second and builds [pprof](https://github.com/google/pprof) formatted profiles from the extracted data. For in-depth detail and explanation refer to the the [Parca Agent Design](/docs/parca-agent-design) documentation.
```

Comment:
Perhaps we can be even more accurate and say 19hz per CPU core. Previously it was 100hz per process, but now it's per CPU core.

Reply:
I added a commit with `per logical CPU`. I find "core" ambiguous as CPU makers typically have a different meaning for it.